persistent-qq/src/Database/Persist/Sql/Raw/QQ.hs

```diff
@@ -69,6 +70,21 @@ parseStr a ('^':'{':xs) = Literal (reverse a) : parseHaskell TableName  [] xs
 parseStr a ('@':'{':xs) = Literal (reverse a) : parseHaskell ColumnName [] xs
 parseStr a (x:xs)       = parseStr (x:a) xs
 
+interpolateValues :: PersistField a => NonEmpty a -> (Text, [[PersistValue]]) -> (Text, [[PersistValue]])
+interpolateValues xs =
+    first (mkPlaceholders values <>) .
+    second (NonEmpty.toList values :)
+    where
+    values = NonEmpty.map toPersistValue xs
+
+mkPlaceholders :: NonEmpty a -> Text
+mkPlaceholders values = "(" <> n `timesCommaSeparated` "?" <> ")"
+    where
```

Comment:
```suggestion
  where
```

Reply:
@parsonsmatt I followed the odd indentation for `where` to match the style used in the rest of the file.  I pushed 
e12934c to fix it everywhere within that source file.  Feel free to cherry-pick.

Reply:
Ah! Thank you. I appreciate it <3